### PR TITLE
added toggle action

### PR DESCRIPTION
**Pull request by Percy-Ikana**

### CODE REVIEW

File: files/manifest.json [hunk 1]
```diff
@@ -32,6 +32,16 @@
 			"UUID": "com.amansprojects.starterpack.runcommand",
 			"PropertyInspectorPath": "propertyInspector/runCommand.html"
 		},
+		{
+			"Icon": "icons/runCommand",
+			"Name": "Run command toggle",
+			"States": [ { "Image": "actionDefaultImage", "Title": "State 1", "ShowTitle": true }, { "Image": "actionDefaultImage", "Title": "State 2", "ShowTitle": true } ],
+			"Controllers": [ "Keypad" ],
+			"SupportedInMultiActions": true,
+			"Tooltip": "Run a command with 2 toggable states",
```
Comment (nekename) [inline files/manifest.json]: Typo, "toggleable" (it's not a real word, but it should at least have an L, and it seems that "LE" is more popular)

File: files/propertyInspector/commandToggle.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	inActionInfo = JSON.parse(inActionInfo);
+
+	websocket.onopen = () => {
+		websocket.send(JSON.stringify({
+			event: inRegisterEvent,
+			uuid: inPropertyInspectorUUID
+		}));
+	};
+
+	let down = document.getElementById("S1Down");
```
Comment (nekename) [inline files/propertyInspector/commandToggle.js]: I would appreciate if you kept these element IDs lowercase, as is the convention. Also, if it's `down2` and `up2`, the first set should be `down1` and `up1`. Just nitpicking though.

**Review by nekename — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just a few minor things. Please also make sure you've run `cargo fmt` and `cargo clippy` before I merge.

File: files/manifest.json (hunk 1 above)
Comment (Percy-Ikana) [reply files/manifest.json]: No I just cant spell =)

**Comment by nekename** [comment]
LGTM, thanks! :+1:

**Pull request merged**